Makefile

```
@@ -10,4 +10,11 @@ linux:
 	GOOS=linux GOARCH=amd64 go build -ldflags="$(FLAGS)" -o bin/ecs-upload-task-linux-$(VERSION) .
 
 windows:
-	GOOS=windows GOARCH=amd64 go build -ldflags="$(FLAGS)" -o bin/ecs-upload-task-win-$(VERSION) .
\ No newline at end of file
+	GOOS=windows GOARCH=amd64 go build -ldflags="$(FLAGS)" -o bin/ecs-upload-task-win-$(VERSION) .
+
+release-multi-arch:
```

You could do this if you want to make sure `clean` is run each time this is executed, and then users don't need to run it.

```suggestion
release-multi-arch: clean
```